src/styles/page.scss

```diff
@@ -0,0 +1,15 @@
+html {
```

Comment:
It's recommended to apply font styles to the body tag instead of the html tag.

Reply:
all requirements done. :)